### PR TITLE
jc: add page

### PR DESCRIPTION
**Pull request by kellyjonbrazil**
jc command (https://github.com/kellyjonbrazil/jc)

- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/common/jc.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# jc
+
+> A utility to convert the output of many commands to JSON.
+> More information: <https://github.com/kellyjonbrazil/jc>.
+
+- Convert command output to JSON via pipe:
+
+`ifconfig | jc --ifconfig`
```
Comment (einverne) [inline pages/common/jc.md:8]: I think `ifconfig` here can be changed right. If so, I think we should use bracket to indicate that this part can be change.

```suggestion
`{{ifconfig}} | jc {{--ifconfig}}`
```

Comment (kellyjonbrazil) [reply pages/common/jc.md:8]: Ah yes, that works!

File: pages/common/jc.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# jc
+
+> A utility to convert the output of many commands to JSON.
```
Comment (zdroid) [inline pages/common/jc.md:3]:
```suggestion
> A utility to convert the output of multiple commands to JSON.
```

**Comment by andrik** [comment]
Thanks for your contribution @kellyjonbrazil and welcome to the project

**Pull request merged**